src/frontend/src/components/Post/telescope-post-content.css

```diff
@@ -82,3 +82,16 @@
 .telescope-post-content a:visited {
   color: #35001d;
 }
+
+.telescope-post-content blockquote,
+.telescope-post-content q {
+  font: 1.5rem/1.5rem Palatino, Georgia, Cambrai, 'Times New Roman', Times, 'Helvetica Neue',
```

Comment:
I don't think this is helping the readability.  Here it is applied:

<img width="744" alt="Screen Shot 2020-04-06 at 8 39 17 AM" src="https://user-images.githubusercontent.com/427398/78559285-21b44d00-77e2-11ea-892e-9a5ea0b79f92.png">

And if I remove this `font` rule:

<img width="748" alt="Screen Shot 2020-04-06 at 8 43 26 AM" src="https://user-images.githubusercontent.com/427398/78559629-b7e87300-77e2-11ea-8f59-ccb0035f60d8.png">

Reply:
Ok. I will remove font rule